### PR TITLE
Add intro page for the game

### LINKED ISSUE #23: Startscreen / Intro of the game
# User Story
As a player
I need to know whats the game is about and which keys I can use (another issue)

# Description
- floating popup screen?
- little storytelling intro
- keys to use
- keys should look like, like they are pressed

# Material

# To do's:
- [x] write intro-text
- [x] new component
- [x] add images of keys
- [x] add "weiter"-key

### PR DESCRIPTION
**Pull request by JoscaWij**
Fixes #23 
introPage deployment: https://spacey-gameintro-cwwonjte4kdsc.herokuapp.com/game/intro
storybook: https://spacey-gameintro-cwwonjte4kdsc.herokuapp.com/storybook/?path=/story/game-intro--game-intro

💭 the user should know which keys to use to play
💭 styling + added to storybook
💭 routing to page and then to game
💭 go back button (different pullrequest. will be added later in the top left corner)

### CODE REVIEW

**Comment by JoscaWij** [comment]
image

File: client/src/components/OutlinedContainer.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  background-image: linear-gradient(
+    180deg,
+    var(--highlight-color),
+    transparent
+  );
+`;
+
+function OutlinedContainer({ children }) {
+  return <OutlinedDiv>{children}</OutlinedDiv>;
+}
```
Comment (LebkuchenCJ) [inline client/src/components/OutlinedContainer.js]: If you only export `{children}` you could think of removing this block as the styled component will pass children already.

File: client/src/page/IntroPage.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+function IntroPage() {
+  return (
+    <IntroPageWrapper>
+      <OutlinedContainer>
+        <IntroTextbox>
+          <span>Please help!</span>
+          <p>
+            The little astronaut is stranded on a lonley planet. He needs to get
+            back to his spaceship as soon as possible.
```
Comment (LebkuchenCJ) [inline client/src/page/IntroPage.js]: This poor astronaut...
image

File: client/src/page/IntroPage.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    text-align: center;
+    flex-basis: 100%;
+  }
+`;
+
+const ImageCenteredContainer = styled.div`
+  background: transparent;
+  display: flex;
+
+  *:first-child {
```
Comment (LebkuchenCJ) [inline client/src/page/IntroPage.js]: I think lint is nagging here. You could think of using `:first-of-type`, too

**Review by LebkuchenCJ — COMMENTED** [review]
NICE JOB. 👍 💯 
Only some small comments. 
One thing I noticed: You could add your milestones to your projects to keep track of your sprints

**Pull request merged**